app/models/institution.rb

```diff
@@ -153,6 +153,8 @@ class Institution < ActiveRecord::Base
   validates :version, :institution, :country, presence: true
   validates :institution_type_name, inclusion: { in: TYPES }
 
+  has_many :yellow_ribbon_programs
```

Comment:
adding dependent destroy here makes sense I believe, unless this should really be a `has_many :through` relationship with a join table?

Reply:
👍 should be good for the dependent destroy.